ui/tests/integration/components/mfa/method-form-test.js

```diff
@@ -0,0 +1,77 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, fillIn, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+module('Integration | Component | mfa-method-form', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    this.store = this.owner.lookup('service:store');
+    this.model = this.store.createRecord('mfa-method');
+    this.model.type = 'totp';
+  });
+
+  test('it should render correct fields', async function (assert) {
+    await render(hbs`
```

Comment:
Can we put `assert.expect` on this test too please?

Reply:
Sure. 👍